packages/python/accern_xyme/util.py

```diff
@@ -206,7 +207,8 @@ def has_graph_easy() -> bool:
 
     try:
         import subprocess
-        subprocess.Popen(["graph-easy", "--help"])
+        subprocess.Popen(  # pylint: disable=consider-using-with
```

Comment:
also new line after import -- what are we doing with the process? we just keep it undone? can you add a FIXME -- this is an issue